### PR TITLE
sequence diagrams using ascii arts

### PR DESCRIPTION
**Pull request by peppelinux**
This PR replace the sequence diagrams previously made using the mermaid notation with ascii art sequence diagrams

### CODE REVIEW

**Review by selfissued — APPROVED** [review]

**Comment by selfissued** [comment]
If these diagrams were generated with a tool, it would be useful to have a link to the diagram sources in comments (like we did in the Federation spec).  According to https://stackoverflow.com/questions/4823468/comments-in-markdown, you can create Markdown comments with the syntax:

> (empty line)
> [comment]: # (This actually is the most platform independent comment)

File: openid-federation-wallet-1_0.md [hunk 1]
```diff
@@ -280,6 +280,8 @@ This section defines the Entity Types used by Organizational Entities in their E
 | Authorization Server  | `federation_entity`, `oauth_authorization_server`          | [@!OpenID4VCI], [@!RFC8414]                    |
 | Credential Issuer     | `federation_entity`, `openid_credential_issuer`, `oauth_authorization_server` | [@!OpenID4VCI], this specification |
 | Credential Verifier         | `federation_entity`, `openid_credential_verifier`         | [@!OpenID.Federation], [@!OpenID4VP], this specification       |
+**Table 1**: Federation Trust Chain and Wallet Attestation are separate things, where the Wallet Attestation is linked to its Issuer attested within the Trust Chain.
```
Comment (peppelinux) [inline openid-federation-wallet-1_0.md:283]:
```suggestion
**Table 1**: Map of the Federation Entity Types and corresponding metadata types for the Wallet architectures.
```

File: openid-federation-wallet-1_0.md [hunk 2]
```diff
@@ -279,8 +279,8 @@ This section defines the Entity Types used by Organizational Entities in their E
 | Wallet Provider       | `federation_entity`, `openid_wallet_provider`              | this specification                                  |
 | Authorization Server  | `federation_entity`, `oauth_authorization_server`          | [@!OpenID4VCI], [@!RFC8414]                    |
 | Credential Issuer     | `federation_entity`, `openid_credential_issuer`, `oauth_authorization_server` | [@!OpenID4VCI], this specification |
-| Credential Verifier   | `federation_entity`, `openid_credential_verifier`         | [@!OpenID.Federation], [@!OpenID4VP], this specification       |
-
+| Credential Verifier   | `federation_entity`, `openid_credential_verifier`          | [@!OpenID.Federation], [@!OpenID4VP], this specification       |
+**Table 1**: Map of the Federation Entity Types and corresponding metadata types for the Wallet architectures.
```
Comment (peppelinux) [inline openid-federation-wallet-1_0.md:283]:
```suggestion
**Table 1**: Map of the Federation Entity Types and corresponding metadata types for the Wallet architectures.


```

File: openid-federation-wallet-1_0.md [hunk 3]
```diff
@@ -465,60 +466,117 @@ Wallets begin by discovering the identity of Credential Issuers through the fede
...
-    Wallet->>Wallet: Extract Authority Hints from CI's Configuration
-
-    loop for each Authority Hint
-        Wallet->>IE: Fetch Entity Configuration
-        Wallet->>IE: Fetch Subordinate Statement
-        Wallet->>Wallet: Validate the previous statement<br/>using the Federation Entity Keys<br/>provided in the Subordinate Statement
-    end
-
-    Wallet->>Wallet: Validate Trust Chain
+The diagram above shows how a Wallet navigates the federation, collecting and validating the Trust Chain for each Credential Issuer (CI), and creating a discovery page including each Credential Issuer using the information, such as the credential types and logo obtained through their Trust Chain.
```
Comment (peppelinux) [inline openid-federation-wallet-1_0.md:511]:
```suggestion
The diagram above shows how a Wallet navigates the federation, collecting and validating the Trust Chain for each Credential Issuer (CI), and creating a discovery page including each Credential Issuer using the information, such as the Credential Types and logo obtained through their Trust Chain.
```

**Pull request merged**